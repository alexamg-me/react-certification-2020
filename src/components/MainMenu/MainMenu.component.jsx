import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  display: inline-block;
  margin: 8px 10px 0 0;
`;
const HamButton = styled.div`
  display: block;
`;
const HamLine = styled.div`
  display: block;
  width: 30px;
  height: 4px;
  margin-bottom: 5px;
  border-radius: 100px;
  background-color: #409287;
`;

function MainMenu() {
  return (
    <Wrapper>
      <HamButton>
        <HamLine />
        <HamLine />
        <HamLine />
      </HamButton>
    </Wrapper>
  );
}

export default MainMenu;
