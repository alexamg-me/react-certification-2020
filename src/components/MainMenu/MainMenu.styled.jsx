import styled from 'styled-components';

export const Wrapper = styled.div`
  display: inline-block;
  margin: 8px 10px 0 0;
`;
export const HamButton = styled.div`
  display: block;
`;
export const HamLine = styled.div`
  display: block;
  width: 30px;
  height: 4px;
  margin-bottom: 5px;
  border-radius: 100px;
  background-color: ${(props) => props.theme.primary_shadow || 'red'};
  transition: all 0.4s ease-in-out;
`;
