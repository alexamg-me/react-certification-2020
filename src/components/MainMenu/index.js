export { default } from './MainMenu.component';
