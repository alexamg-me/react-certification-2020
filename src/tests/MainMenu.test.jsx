import React from 'react';
import TestRenderer from 'react-test-renderer';
import MainMenu from '../components/MainMenu/MainMenu.component';

describe(`Component: MainMenu`, () => {
  test(`MainMenu renders with default props`, () => {
    const component = TestRenderer.create(<MainMenu />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
