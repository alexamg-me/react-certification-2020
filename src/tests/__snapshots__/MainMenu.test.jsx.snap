// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: MainMenu MainMenu renders with default props 1`] = `
<div
  className="sc-bdfBwQ kPShWq"
>
  <div
    className="sc-gsTCUz dtQtlu"
  >
    <div
      className="sc-dlfnbm ieMrRo"
    />
    <div
      className="sc-dlfnbm ieMrRo"
    />
    <div
      className="sc-dlfnbm ieMrRo"
    />
  </div>
</div>
`;
